automata/fa/dfa.py

```diff
@@ -229,7 +227,7 @@ def _check_for_input_rejection(self, current_state):
                 'the DFA stopped on a non-final state ({})'.format(
                     current_state))
 
-    def read_input_stepwise(self, input_str):
+    def read_input_stepwise(self, input_str, allow_rejection=False):
```

Comment:
@Tagl The default for `allow_rejection` should actually be `True` in order to preserve backwards compatibility (since that is the current behavior in v6.x).